src/test/java/picard/util/AlleleSubsettingUtilsTest.java

```diff
@@ -19,6 +19,7 @@
     final List<Allele> AA = Arrays.asList(Aref, Aref);
     final List<Allele> AC = Arrays.asList(Aref, C);
     final List<Allele> CC = Arrays.asList(C, C);
+    final List<Allele> noCall = Arrays.asList(Allele.NO_CALL, Allele.NO_CALL);
```

Comment:
Likewise, this is `GATKVariantContextUtils.NO_CALL_ALLELES`.

Reply:
https://youtu.be/nIaN9Koa9oM not Kansas anymore... :-)

Reply:
Then bring Topeka to Oz.